lib/cog_api/fake/relay_groups.ex

```diff
@@ -114,13 +119,18 @@ defmodule CogApi.Fake.RelayGroups do
     update_bundles(bundle_with_group, relay_group)
   end
 
-  def remove_bundle(_, _, %Endpoint{token: nil}), do: Endpoint.invalid_endpoint
-  def remove_bundle(%{name: name}, %{bundle: bundle_name}, %Endpoint{token: _}) do
-    bundle = Server.show_by_key(Bundle, :name, bundle_name)
+  def remove_bundles(_, _, %Endpoint{token: nil}), do: Endpoint.invalid_endpoint
+  def remove_bundles(%{name: name}, %{bundles: bundle_names}, %Endpoint{token: _}=endpoint) do
     relay_group = Server.show_by_key(RelayGroup, :name, name)
-    remove_bundle(bundle, relay_group)
+    bundle_ids = Enum.map(bundle_names, &Server.show_by_key(Bundle, :name, &1))
+    |> Enum.map(&Map.fetch!(&1, :id))
+    remove_bundles(relay_group.id, bundle_ids, endpoint)
+  end
+  def remove_bundles(id, bundle_ids, %Endpoint{token: _}=endpoint) when is_list(bundle_ids) do
+    Enum.map(bundle_ids, &remove_bundles(id, &1, endpoint))
+    |> List.last
   end
-  def remove_bundle(id, bundle_id, %Endpoint{token: _}) do
+  def remove_bundles(id, bundle_id, %Endpoint{token: _}) do
```

Comment:
It looks like the `remove_bundles` without just a list of `bundle_ids` wouldn't work. Is that correct?